Added alpha documentation alpha-v0.4.403 (#55)
oakestra/documentation: content/version/alpha-v0.4.403/docs/getting-started/oak-environment/add-a-cluster.md

---
title: "Add a Cluster"
summary: ""
draft: true
weight: 102040000
toc: true
seo:
  title: "" # custom title (optional)
  description: "" # custom description (recommended)
  canonical: "" # custom canonical URL (optional)
  noindex: false # false (default) or true
---

### Add a New Cluster to your Infrastructure.

> If you want to create new Clusters to attatch to you Oakestra Root, on each Cluster Orchestrator machine you can run:

Reviewer: ```suggestion
If you want to create new Clusters to attach to your Oakestra Root, on each Cluster Orchestrator machine, you can run:
```

```bash
export OAKESTRA_VERSION=alpha-v0.4.403
curl -sfL oakestra.io/install-cluster.sh | sh -
```

This will start a new **Cluster Orchestrator** component. 
A script will ask you for your 
 - **Cluster name** : A name of your choice for this cluster. 
 - **Location** : Geographical coordinates and competence radius in meters, e.g.:`48.1374,11.5755,1000`
 - **IP address of the Root Orchestrator**

{{< callout context="note" title="Note" icon="outline/info-circle" >}}

> All these variables can be set before startup exporting them:

Reviewer: ```suggestion
All these variables can be set before startup, exporting them:
```

```bash
export CLUSTER_LOCATION=<latitude>,<longitude>,<radius>
export CLUSTER_NAME=my_awesome_cluster
export SYSTEM_MANAGER_URL=<url or ip>
```
{{< /callout >}}
{{< callout context="note" title="Note" icon="outline/info-circle" >}}
Run this export before the cluster startup to deploy the Development version of the Cluster Orchestrator

```bash
export OAKESTRA_BRANCH=develop
```

{{< /callout >}}

> You can attatch new worker nodes to this cluster using the same procedure shown in [Setup Your First Cluster](#setup-your-first-cluster)

Reviewer: ```suggestion
You can attach new worker nodes to this cluster using the same procedure shown in [Setup Your First Cluster](#setup-your-first-cluster)
```
